src/topology/continuous_function/zero_at_infty.lean

```diff
@@ -283,9 +283,27 @@ instance {R : Type*} [semiring R] [non_unital_non_assoc_semiring β] [topologica
     rw [←smul_eq_mul, ←smul_eq_mul, smul_comm],
   end }
 
-
 end algebraic_structure
 
+section uniform
+
+variables [uniform_space β] [uniform_space γ] [has_zero γ]
+  [zero_at_infty_continuous_map_class F β γ]
+
+lemma uniform_continuous (f : F) : uniform_continuous (f : β → γ) :=
```

Comment:
Instead of proving the lemma for a `zero_at_infty_continuous_map_class`, could you first prove it for a single function which is continuous and tends to zero at infinity, and then deduce the class version from the concrete version?